autosynth/change_pusher.py

```diff
@@ -148,7 +152,7 @@ def check_if_pr_already_exists(self, branch) -> bool:
         return self.inner_change_pusher.check_if_pr_already_exists(branch)
 
 
-def build_pr_body(synth_log: str):
+def build_pr_body(synth_log: str, trailers: str):
```

Comment:
nit: could possibly default to empty. Later invocations look kinda weird with `build_pr_body(some_var, "")` and make you wonder what the second parameter is.

Reply:
Done.